framework/log/Logger.php

```diff
@@ -263,27 +264,29 @@ public function getDbProfiling()
      * Calculates the elapsed time for the given log messages.
      * @param array $messages the log messages obtained from profiling
      * @return array timings. Each element is an array consisting of these elements:
-     * `info`, `category`, `timestamp`, `trace`, `level`, `duration`.
+     * `info`, `category`, `timestamp`, `trace`, `level`, `duration`, `memory`, `memoryDiff`.
      */
     public function calculateTimings($messages)
     {
         $timings = [];
         $stack = [];
 
         foreach ($messages as $i => $log) {
-            list($token, $level, $category, $timestamp, $traces) = $log;
-            $log[5] = $i;
+            list($token, $level, $category, $timestamp, $traces, $memory) = $log;
```

Comment:
this will break on old logging data, when memory is not set. Will fix that while merging.